src/features/home/components/modules/OffersModule.native.test.tsx

```diff
@@ -158,6 +158,20 @@ describe('OffersModule', () => {
       })
     })
   })
+
+  it('should trigger logEvent "ConsultOffer" when clicking on offer', () => {
+    renderOffersModule({ ...props })
```

Comment:
normalement les composants ne mutent pas leurs props

il est donc pas nécessaire de déconstruire / reconstruire l'objet

```suggestion
    renderOffersModule(props)
```